agenda-webapps/src/main/webapp/vue-app/agenda/js/CalendarService.js

```diff
@@ -29,4 +29,41 @@ export function saveCalendar(calendar) {
       throw new Error('Error updating calendar');
     }
   });
+}
+
+export function saveAgendaSetting(contextKey, contextValue, scopeKey, scopeValue, key, value) {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/settings/${contextKey},${contextValue}/${scopeKey},${scopeValue}/${key}`, {
+    method: 'PUT',
+    credentials: 'include',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      value: value,
+    }),
+  }).then(resp => {
+    if (!resp || !resp.ok) {
+      throw new Error('Response code indicates a server error', resp);
+    }
+  });
+}
+
+export function getAgendaSetting() {
```

Comment:
`getAgendaSettings` instead to say that this is about multiple settings

Reply:
okay